includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -61,6 +61,43 @@ public function __construct( $service ) {
 		);
 	}
 
+	/**
+	 * Determine if the current user can access the feature
+	 *
+	 * @param {string} $feature       Feature to check.
+	 * @param {array}  $feature_roles Roles that have access to the feature.
+	 *
+	 * @return bool
+	 */
+	public function user_can_access( $feature, $feature_roles = [] ) {
+		$access   = false;
+		$settings = $this->get_settings();
+
+		if (
+			! empty( $settings ) &&
+			isset( $settings['authenticated'] ) &&
+			false !== $settings['authenticated'] &&
```

Comment:
I would suggest we remove the authentication checks here.

1. Seems to go against what this function says it does (checking if a user can access)
2. And we recently introduced an `is_configured` method that does this

Reply:
@dkotter, The fix for this is now ready for review. Could you point me in the right direction for the `is_configured` 
 method? This needs to be added as well. Thanks!

cc @jeffpaul